engine/storage/integration-test/src/test/java/com/cloud/vm/snapshot/dao/VmSnapshotDaoTest.java

```diff
@@ -41,7 +39,7 @@
 
     @Test
     public void testVmSnapshotDetails() {
-        VMSnapshotDetailsVO detailsVO = new VMSnapshotDetailsVO(1L, "test", "foo", display);
+        VMSnapshotDetailsVO detailsVO = new VMSnapshotDetailsVO(1L, "test", "foo", true);
```

Comment:
why not use 'display' here?

Reply:
I have no idea why sometimes it compiles. It should not per Java specs. You cannot have a `package` declaration in `x.y.z` and then put this same class in a folder structure as `x.y.z.z`. 

The thing called `display` there is nothing. It is not a variable, it is nothing... We need a boolean variable/value here. This looks like something that eclipse generated when using the refactor feature.

Reply:
@rafaelweingartner there is no hard requirements on the file location. But the compiler will put the corresponding `class` file in the correct directory structure, that's all.

Reply:
I was under the impression that there was a hard requirement. Anyways, I only found the problem because of Eclipse. It was complaining about these files that were in the wrong folder structure.